modules/ROOT/pages/bolt/handshake.adoc

```diff
@@ -105,13 +105,13 @@ The range cannot span multiple major versions.
 
 .Example with versions 4.3 plus two previous minor versions, 4.2 and 4.1
 ----
-00 02 03 04
+01 02 03 04
 ----
 
 .Example where the client is aware of five Bolt versions; 3, 4.0, 4.1, 4.2 and 4.3, and the server responds with 4.1
 ----
 C: 60 60 B0 17
-C: 00 03 03 04 00 00 01 04 00 00 00 04 00 00 00 03
+C: 00 02 03 04 00 00 01 04 00 00 00 04 00 00 00 03
```

Comment:
This change would mean the client requests `[4.3-4.1, 4.1, 4.0, 3.0]`, which is equivalent to the current `[4.3-4.0, 4.1, 4.0, 3.0]`. In fact 
```
C: 00 01 03 04 00 00 01 04 00 00 00 04 00 00 00 03
```
would be another equivalent client request.

I don't see a reason to change it, though. Finally, if we change it, we must not forget to update the note below explaining the versions requested.

Reply:
Sorry for my clumsiness, but I can't understand why this "00 03 03 04" is equivalent to this: 4.3-4.1. Thank you very much.

Reply:
No worries. I might have expressed myself unclear. Let me try again.

As per the docs, a version request of the form
```
00 02 03 04
```
means  Version `4.3` + the 2 consecutive versions before that, i.e., `4.1, 4.2, 4.3` = `4.1-4.3`.
That's obviously not equivalent to
```
00 03 03 04
```
which means `4.3-4.0`. However, with the following versions requested (the whole things reads):
```
C: 00 01 03 04   00 00 01 04   00 00 00 04   00 00 00 03
```
which translates to (parentheses to mark requested ranges): "server, please give me (4.3, 4.2), 4.1, 4.0 or 3.0 (is descending preference)".

Because of the overlap of the range with the explicitly requested versions, this is equivalent to
```
C: 00 03 03 04   00 00 01 04   00 00 00 04   00 00 00 03
```
which translates to: "server, please give me (4.3, 4.2, 4.1, 4.0), 4.1, 4.0 or 3.0 (is descending preference)".

Reply:
Now I understand, I don't know why I always thought it was one version per byte. Now I have it clear, thank you very much.